static_analyzer/scanner.py

```
@@ -27,15 +77,38 @@ def scan(self) -> list[ProgrammingLanguage]:
         """
 
         commands = get_config("tools")["tokei"]["command"]
-        result = subprocess.run(commands, cwd=self.repo_location, capture_output=True, text=True, check=True)
+        try:
+            result = subprocess.run(commands, cwd=self.repo_location, capture_output=True, text=True, check=True)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)